components/Shared/TopNav.tsx

```diff
@@ -0,0 +1,73 @@
+import Logo from "@/public/assets/icons/dashboard/Logo.svg";
+import {
+	ArrowDownIcon,
+	MenuIcon,
+	NotificationIcon,
+	PlusButtonIcon,
+} from "@/public/assets/icons/dashboard/icons";
+import userIcon from "@/public/assets/icons/dashboard/userIcon.svg";
+import Image from "next/image";
+import Link from "next/link";
+import { Button } from "../ui";
+
+type TopNavProps = {
+	toggleSidebar: () => void;
+};
+
+export default function TopNav({ toggleSidebar }: TopNavProps) {
+	return (
+		<div className="sticky top-0 flex items-center justify-between border-b border-gray-400 p-7  md:border-gray-300 md:p-4 md:pr-10">
+			<div className="hidden md:block">
+				<h1 className="text-sm font-semibold"> Hi, Locs Designer👋</h1>
+				<p className="text-sm">Here is an overview of your campaign ✨</p>
+			</div>
+			<div className="flex items-center justify-center gap-3 md:hidden">
+				<div className="cursor-pointer" onClick={toggleSidebar}>
+					<MenuIcon />
+				</div>
+				<Link href={"/dashboard"}>
+					<Image
+						src={Logo as string}
+						width={28}
+						height={28}
+						alt="abeghelp logo"
+					/>
+				</Link>
+			</div>
+			<div className="flex items-center md:space-x-3">
+				<div className="hidden md:block">
+					<Button className="flex items-center rounded-md bg-teal-700 p-2">
+						<span className="pr-2">
+							<PlusButtonIcon />
+						</span>
+						Create Campaign
+					</Button>
+				</div>
+				<div className="relative mr-1 md:mr-0">
+					<div className="absolute left-0 top-3 hidden h-5 border-l border-gray-400 md:block"></div>
```

Comment:
instead of doing this, use the tailwind divide property on the parent element so that it will add border in between the children. search the tailwind docs to see how it is used and make changes